fenix/app/src/main/java/org/mozilla/fenix/settings/advanced/LocaleUtils.kt

```diff
@@ -0,0 +1,232 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+package org.mozilla.fenix.settings.advanced
```

Comment:
Let's create a `fenix.locale` package or preferable see if there's a top level `utils`-like package already to not hide this within `settings.advanced` since we are using this more broadly.

Reply:
I moved the class in org.mozilla.fenix.utils